tests/crud/test_case.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for the CRUD module."""
+from arnold.adapter import ArnoldAdapter
+from arnold.crud.create import create_case
+from arnold.crud.read.case import get_case
+from arnold.models.database.case import Case
+
+
+def test_create_case(mock_adapter: ArnoldAdapter, balsamic_case_json: dict):
+    """Test creating a case in the database."""
+    # GIVEN a case document
+    case: Case = Case.model_validate(balsamic_case_json)
+
+    # WHEN creating a case in the database
+    create_case(case=case, adapter=mock_adapter)
+
+    # THEN a case is added
+    new_case: Case = mock_adapter.case_collection.find_one()
```

Comment:
Would love some more specific way of fetching this than getting the first available case.